tools/MarkdownConverter/Spec/MarkdownSpec.cs

```diff
@@ -247,6 +247,9 @@ private static string RemoveBlockComments(string text, string file)
         /// them as separate spans. We remove this later on when converting Markdown to Word. Note that this
         /// only works at the top level at the moment.
         /// </summary>
+        /// <remarks>
+        /// This may no longer be necessary now that we're using Markdown Lint to avoid this condition.
```

Comment:
I think it probably *is* still necessary, when they're in separate paragraphs. For example:

```markdown
> *Example*:
>
> Some text in multiple paragraphs.
>
> *end example*

> *Note*:
>
> Some text in multiple paragraphs.
>
> *end note*
```

I *think* the two will be collapsed so we end up with an "end example Note:" paragraph without this. I can add a test for it once this PR is in, perhaps.